README.md

```diff
@@ -81,8 +89,8 @@ CMake is used to generate our project files, if you haven't used it before we wi
   ```
 
   Now, you will be able to open the solution, and build it.
-
-
+  ##NOTE## You will have cd build and cmake again if you add any files or you will get an error when building. //is this also required on the visual studio extension?
```

Comment:
Yes it's always required, Visual Studio, VS Code or any other IDE.
CMake instructs our compiler which files should be compiled.